Autopilot/PathManager/Src/landingTouchdownStage.cpp

```diff
@@ -0,0 +1,22 @@
+#include "pathStateClasses.hpp"
+
+void landingTouchdownStage::execute(pathManager* pathMgr)
+{
+    //throttleOff()
+    //aligning heading
+    cruisingState::_outputdata.desiredHeading = getFromTelemetry::telemetryInput.landingDirection;
```

Comment:
why do you need to modify cruising state variables from here ? I'm weary about doing this since it can lead to spaghetti code.